### DISCUSSION TITLE
Question regarding perspective when selecting points using CutPlotter tool

### DISCUSSION THREAD

**Discussion by jkissi** [marcomusy/vedo#712] [Q&A]
Hi Marco, hope you are well!
I have a quick question about zooming when I am using the plotter() object. It seems that when I try and zoom into my points in the plotter window, the closer I get to the points, the less I am able to zoom, until I cannot zoom any further. Problem is, that the point at which I cannot zoom any further is a very long way from the points, and the cutplotter I have built requires that you select points from within a created spline. This perspective difference can be a problem. The picture below shows as far as I can zoom and the yellow points indicate what has actually been selected. So, you can see how far away from the points the camera has stopped. 

![perspective_zoom](https://user-images.githubusercontent.com/8246894/197296340-81f56031-7df9-46e9-a4cb-aef097566538.PNG)

Can you think of a way to change the zoom behaviour so I can get closer? Or modify something so that all the points within the boundary are always selected, without correcting for perspective?

Once again, cheers Marco!!!
j

**Comment by marcomusy** [comment]
Hi J.
instead of zooming have you tried to set the camera position and focus explicitly (press "C" for a template)?
Are you using parallel or perspective projections (toggle by pressing "u")?
